src/ResourceManager/Resources/ChangeLog.md

```diff
@@ -21,6 +21,7 @@
 
 ## Version 6.1.0
 * Fix issue with `Properties` property of `PSResource` object(s) returned from `Get-AzureRmResource`
+* Fix issue with error message(s) not being returned for ManagedApplication and ManagedApplicationDefinition cmdlets
```

Comment:
@vivsriaus looks like your update got moved to under the `6.1.0` header with the merge -- you can move your line to be under `Current Release`